packages/connect/src/types/api/ethereumCall.ts

```diff
@@ -0,0 +1,6 @@
+import type { CommonParamsWithCoin, Params, Response } from '../params';
+import type { EthereumCall } from './ethereum';
+
+export declare function ethereumCall(
+    params: CommonParamsWithCoin & Params<EthereumCall>,
+): Response<{ data: string }>;
```

Comment:
guys, shouldn't this be named `blockchainRpcCall`? 
similar logic to for example `blockchainEstimateFee`? 

cc @martykan @marekrjpolak

Reply:
or maybe blockchainEthRpcCall ?

Reply:
It's true that the coin methods usually do something with the device, this method just talks to blockbook like the rest of the blockchain methods. 
`blockchainEthRpcCall` or `blockchainEvmRpcCall` sounds good

Reply:
@tomasklim 
So what naming we should apply?

Reply:
lets do `blockchainEvmRpcCall`

Reply:
renamed, https://github.com/trezor/trezor-suite/pull/12122/commits/a854667e5b7374e48206cbdb4033a7ed6f2eb5f7